src/views/registration-js/attendees/fields/birth.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
+ *
+ * @version TBD
+ *
+ * @see     Tribe__Tickets_Plus__Meta__Field__Birth
+ *
+ */
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id  = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$field      = (array) $field;
+$field_name = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled   = false;
+$classes    = [ 'tribe-common-b1', 'tribe-field', 'tribe-tickets__item__attendee__field__birth' ];
+if ( $required ) {
+	$classes[] = 'tribe-tickets-meta-required';
+}
+$days   = Tribe__Tickets_Plus__Meta__Field__Birth::get_days();
+$months = Tribe__Tickets_Plus__Meta__Field__Birth::get_months();
+$years  = Tribe__Tickets_Plus__Meta__Field__Birth::get_years();
+?>
+<div <?php tribe_classes( $classes ); ?> >
+	<label
+			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+			for="<?php echo esc_attr( $option_id ); ?>"
+	><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+
+	<div class="tribe__tickets__item__attendee__field__birth__month">
+		<select
+				<?php tribe_disabled( $disabled ); ?>
+				<?php tribe_required( $required ); ?>
+				id="tribe-events-birth-date-month"
+		>
+			<option value="" disabled selected><?php esc_html_e( "Month", 'tribe-event-plus' ); ?></option>
+			<?php
+			foreach ( $months as $month ) {
+				$month = esc_attr( $month );
+				echo "<option>$month</option>";
+			}
+			?>
+		</select>
+	</div>
+	<div class="tribe__tickets__item__attendee__field__birth__day">
+		<select
+				<?php tribe_disabled( $disabled ); ?>
+				<?php tribe_required( $required ); ?>
+				id="tribe-events-birth-date-day"
+		>
+			<option value="" disabled selected><?php esc_html_e( "Day", 'tribe-event-plus' ); ?></option>
+			<?php
+			foreach ( $days as $day ) {
+				$day = esc_attr( $day );
+				echo "<option>$day</option>";
+			}
+			?>
+		</select>
+	</div>
+	<div class="tribe__tickets__item__attendee__field__birth__year">
+		<select
+				<?php tribe_disabled( $disabled ); ?>
+				<?php tribe_required( $required ); ?>
+				id="tribe-events-birth-date-year"
+		>
+			<option value="" disabled selected><?php esc_html_e( "Year", 'tribe-event-plus' ); ?></option>
+			<?php
+			foreach ( $years as $year ) {
+				$year = esc_attr( $year );
+				echo "<option>$year</option>";
+			}
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '"<option>$year</option>"'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '"<option>$year</option>"'.

<sub>hash: 1a9bf08092d1449941a6f819a4e5c61e</sub>